Userland/Libraries/LibCore/System.h

```diff
@@ -205,6 +209,11 @@ ErrorOr<void> exec_command(Vector<StringView>& command, bool preserve_env);
 
 ErrorOr<void> exec(StringView filename, ReadonlySpan<StringView> arguments, SearchInPath, Optional<ReadonlySpan<StringView>> environment = {});
 
+#ifdef AK_OS_SERENITY
+ErrorOr<u32> create_unshare(Kernel::UnshareType type, unsigned flags);
+ErrorOr<void> attach_unshare(Kernel::UnshareType type, unsigned index);
```

Comment:
Probably nicer for searchability if these are `unshare_create` and `unshare_attach` respectively.